src/hosted-buttons/types.js

```diff
@@ -1,8 +1,27 @@
 /* @flow */
 /* eslint-disable no-restricted-globals, promise/no-native */
 
+export type FlexDirection = string;
```

Comment:
I would love to update all the strings types to their actual string values, but I'm thinking that could be a separate ticket once we have the updated response from NCP team